specerror/config-linux.go

```diff
@@ -10,14 +10,21 @@ import (
 const (
 	// DefaultFilesystems represents "The following filesystems SHOULD be made available in each container's filesystem:"
 	DefaultFilesystems = "The following filesystems SHOULD be made available in each container's filesystem:"
+	NamespacesType     = "The following namespace types are supported:"
+	NamespacesPath     = "This value MUST be an absolute path."
 )
 
 var (
 	defaultFilesystemsRef = func(version string) (reference string, err error) {
 		return fmt.Sprintf(referenceTemplate, version, "config-linux.md#default-filesystems"), nil
 	}
+	namespacesRef = func(version string) (reference string, err error) {
+		return fmt.Sprintf(referenceTemplate, version, "config-linux.md#namespaces"), nil
+	}
 )
 
 func init() {
 	register(DefaultFilesystems, rfc2119.Should, defaultFilesystemsRef)
+	register(NamespacesType, rfc2119.Should, namespacesRef)
```

Comment:
The spec says: "The following namespace types are supported", We don't have key words like 'SHOULD'. Maybe we can add a PR to spec project?

Reply:
>Maybe we can add a PR to spec project?

Approve.

Reply:
I open https://github.com/opencontainers/runtime-spec/pull/931.